Source/Core/Core/PowerPC/JitArm64/Jit.cpp

```diff
@@ -276,7 +279,8 @@ void JitArm64::SingleStep()
 
 void JitArm64::Jit(u32)
 {
-	if (GetSpaceLeft() < 0x10000 || blocks.IsFull() || SConfig::GetInstance().bJITNoBlockCache)
+	if (GetSpaceLeft() < 0x10000 || farcode.GetSpaceLeft() < 0x10000 || blocks.IsFull() ||
```

Comment:
Maybe move the ```GetSpaceLeft() < 0x10000``` into a function similar to ```blocks.IsFull()```? That way if the buffer size changes, you don't have to go find all the places to change ```0x10000``` to something else.